jmeter-service/main.go

```diff
@@ -110,7 +122,8 @@ func runTests(event cloudevents.Event, shkeptncontext string, data keptnv2.TestT
 		}
 
 		if !res {
-			if err := sendTestsFinishedEvent(shkeptncontext, event, startedAt, keptnv2.ResultFailed, logger); err != nil {
+			msg := fmt.Sprintf("Workload Testing Completed")
```

Comment:
Please do not set the message to "Workload Testing Completed", but rather derive a meaningful message from the `runWorkload` function. See `res, err = runWorkload(serviceUrl, testInfo, healthCheckWorkload, logger)` a couple lines above.